source/storage/apis-1.5.rst

```diff
@@ -598,22 +598,33 @@ Response Headers
 
     This header gives the last-modified time of the target resource
     as seen during processing of the request, and will be included in all
-    success responses (200, 201, 204).  When given in response to a write
-    request, this will be equal to the server's current time and to the new
-    last-modified time of any BSOs created or changed by the request.
+    success responses (200, 201, 204). It is similar to the standard HTTP
+    **Last-Modified** header, but the value is a decimal timestamp rather than
```

Comment:
Should we specify millisecond precision, or that it's arbitrary precision?

Reply:
updated, thanks